intellijJVersions.gradle

```diff
@@ -12,7 +12,7 @@ static def ideProfiles() {
                             ]
                     ]
             ],
-            "2019.1": [
+            "2019.1": [ // TODO: Marker comment: FIX_WHEN_MIN_IS_192
```

Comment:
Do we need to mark 183 as well?

Reply:
We will probably remove 183 at same time as 191

Reply:
OK. I think one marker is enough then.